src/domain/email/email.repository.ts

```diff
@@ -1,41 +1,23 @@
 import { IEmailDataSource } from '@/domain/interfaces/email.datasource.interface';
 import { Inject } from '@nestjs/common';
-import { ISafeRepository } from '@/domain/safe/safe.repository.interface';
 import codeGenerator from '@/domain/email/code-generator';
 import { Email } from '@/domain/email/entities/email.entity';
 import { IEmailRepository } from '@/domain/email/email.repository.interface';
-import { SignerNotOwnerError } from '@/domain/email/errors/signer-not-owner.error';
 import { EmailSaveError } from '@/domain/email/errors/email-save.error';
 
 export class EmailRepository implements IEmailRepository {
   constructor(
     @Inject(IEmailDataSource)
     private readonly emailDataSource: IEmailDataSource,
-    @Inject(ISafeRepository) private readonly safeRepository: ISafeRepository,
   ) {}
 
   async saveEmail(args: {
     chainId: string;
-    safeAddress: string;
+    safe: string;
```

Comment:
Nitpick: why is this renamed to `safe` internally in this repository? I mean, in the public API (controller) `safeAddress` is used, and it is named `safeAddress` in `EmailDataSource` also, so it feels a bit more straightfoward to keep that name within the flow. And also I would tend to think `safe` variable contains a `Safe` entity.

No strong opinion on this, it's a minor detail 🙂

Reply:
I agree 👍 Renamed in [b141e7b](https://github.com/safe-global/safe-client-gateway/pull/862/commits/b141e7b9baad3c5528d2ed74cf5f2e98cdb0bbb4)

No strong reason for the rename. It was to keep the names short but in the domain it does not make sense since we have a `Safe` entity which I expect to be associated with the name `safe` as you mentioned.

Reply:
I agree 👍 Renamed in [b141e7b](https://github.com/safe-global/safe-client-gateway/pull/862/commits/b141e7b9baad3c5528d2ed74cf5f2e98cdb0bbb4)

No strong reason for the rename. It was to keep the names short but in the domain it does not make sense since we have a `Safe` entity which I expect to be associated with the name `safe` as you mentioned.